geometry/list-transformations-by-suitability/src/main/java/com/esri/samples/list_transformations_by_suitability/ListTransformationsBySuitabilitySample.java

```diff
@@ -100,10 +100,13 @@ public void start(Stage stage) {
       transformationsListView.setCellFactory(list -> new ListCell<DatumTransformation>() {
 
         @Override
-        protected void updateItem(DatumTransformation transformation, boolean bln) {
+        protected void updateItem(DatumTransformation transformation, boolean empty) {
```

Comment:
naming conventions for booleans can be vague with Java, but we can try and be more succinct with their naming here. something like `isEmpty` would be more readable here.
```suggestion
        protected void updateItem(DatumTransformation transformation, boolean isEmpty) {
```

Reply:
@Rachael-E At our review meeting on Friday, the samples team confirmed that we'll remove the colour picking functionality. Setting the colours explicitly in this sample shows off the ability to configure the colours of the viewshed areas without impacting the other samples.